package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "material-design-icons-crawler",
+  "version": "1.0.0",
+  "description": "",
+  "main": "generator.js",
+  "scripts": {
+    "build": "node generator.js",
```

Comment:
Attention, the build command is a reserved command that will be used when you execute "npm version patch/minor/major". That's why I would rename this command to something more explicitly named like "import".

In addition, in this file lots of required dependencies and scripts are missing. They are required for publishing the package. See: https://github.com/aboutbits/pagination/blob/master/package.json

Reply:
All necessary fields have been added. Since tsc is not required for this package, the build command executes the bootstrapping command to create the icon components